docs/RELEASE_NOTES.md

```diff
@@ -146,287 +147,253 @@ Use Cocos Console to create a new game:
 cocos new -l cpp|js|lua MyNewGame
 ```
 
-# v3.9
+# v3.10
 
-## Highlights features, improvements and API updates of v3.9
+## Highlights features, improvements and API updates of v3.10
 
-We are happy to announce the release of Cocos2d-x v3.9. Following are the highlighted features, improvements and API updates in this version. 
+We are happy to announce the release of Cocos2d-x v3.10. Following are the highlighted features, improvements and API updates in this version. 
 
-1. 3D Module: 
-    - Added 3D MotionStreak to support streak effect. 
-    - Refined Sprite3D to support material system. 
-2. 2D Module:
-    - Added frame callback function and animation callback function. 
-    - Added script component system. 
-    - Reconstruction of 2D physics with Component. 
-    - Improved EditBox implemention on iOS and Win32 platform.
-    - Removed dependence of libcurl on AssetsManager, AssetsManagerEx and Downloader (iOS & Android).
-    - Improved particle performance. 
-3. Others: 
-    - Supported Action inheritance, update function overwriting in JSB. 
-    - Improved ScrollView performance in Web engine. 
-    - Improved Scale9Sprite performance in Web engine. 
-    - Decoupled Sprite's setTexture and updateColor in Web engine.
-    - Added support for debugging and release on real devices with Xcode7 and iOS9.
+1. UI System:
+    - Reimplemented Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
+    - Changed PageView to derived from ListView, PageView can be added any widget as child.
+    - Added three overflow type to new label: CLAMP，SHRINK，RESIZE_HEIGHT.
+2. JSModule:
+    - Improved JS bindings with more secured memory management, and enhance the stability of the engine, streamline all JSB code with the GC coding style of SpiderMonkey([https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide "https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide")).
```

Comment:
The link seems to be wrong, suggestion:

[GC coding style of SpiderMonkey](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/GC_Rooting_Guide)